pkg/core/native/native_nep17.go

```diff
@@ -138,7 +143,34 @@ func addrToStackItem(u *util.Uint160) stackitem.Item {
 	return stackitem.NewByteArray(u.BytesBE())
 }
 
-func (c *nep5TokenNative) emitTransfer(ic *interop.Context, from, to *util.Uint160, amount *big.Int) {
+func (c *nep17TokenNative) postTransfer(ic *interop.Context, from, to *util.Uint160, amount *big.Int) {
+	c.emitTransfer(ic, from, to, amount)
+	if to == nil {
+		return
+	}
+	cs, err := ic.DAO.GetContractState(*to)
+	if err != nil {
+		return
+	}
+	// TODO this is missing in C# node
+	if cs.Manifest.ABI.GetMethod(onPaymentName) == nil {
+		return
+	}
+
+	fromArg := stackitem.Item(stackitem.Null{})
+	if from != nil {
+		fromArg = stackitem.NewByteArray((*from).BytesBE())
+	}
+	args := []stackitem.Item{
+		fromArg,
+		stackitem.NewBigInteger(amount),
+	}
+	if err := contract.CallExInternal(ic, cs, onPaymentName, args, smartcontract.All, vm.EnsureIsEmpty); err != nil {
+		panic(err)
+	}
+}
+
+func (c *nep17TokenNative) emitTransfer(ic *interop.Context, from, to *util.Uint160, amount *big.Int) {
 	ne := state.NotificationEvent{
 		ScriptHash: c.Hash,
 		Name:       "Transfer",
```

Comment:
We can make `handleNotification` in Blockchain work only with `Transfer` now. And I guess any other references to NEP5 can be changed now to NEP17 (types/RPC requests). neo-project/neo-modules#405

Reply:
Renamed everything in a separate commit.